src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -74,6 +75,8 @@ internal unsafe IReferenceTrackerVftbl ReferenceTracker
             }
         }
 
+        internal bool CleanupRCW { get; set; }
```

Comment:
I would avoid creation of a boolean field and instead just use the counter.

For example:
```csharp
internal void MarkCleanupRCW() => _rcwCleanupCounter = 2;
private bool CleanupRCW { get => _rcwCleanupCounter == 0; }
```